uast/uast.go

```diff
@@ -33,6 +33,21 @@ const (
 	// File is the root node of a single file AST.
 	File
 
+	// Binary aritmethic operators. Examples with C operators.
+	// TODO: should we have division and FloorDivision like Python or Nim?
+	// TODO: should we had the pow operator that some languages have?
+
+	// OpArithmeticAdd is the binary add operator (i.e. + in most languages)
+	OpArithmeticAdd
+	// OpArithmeticSubstract is the binary subtract operator (i.e. - in most languages)
+	OpArithmeticSubstract
+	// OpArithmeticMultiply is the binary multiply operator (i.e. * in most languages)
+	OpArithmeticMultiply
+	// OpArithmeticDivision is the binary division operator (i.e. / in most languages)
+	OpArithmeticDivision
```

Comment:
About the division: I decided to went with what most languages I checked use for division and remainder in their AST node namings. For example as I wrote in the "TODO", Python3 also have // for integer division. Quotient would just be the one of the two subnode literal/symbol/expressions, we could add it as a role but I don't think is really needed.